src/economy/shops/messages.ts

```diff
@@ -0,0 +1,244 @@
+import { addTranslation } from '../../common/localization/localization';
+import { ChatColor } from 'org.bukkit';
+import { color, text } from 'craftjs-plugin/chat';
+
+export const SHOP_YELLOW = '#FFFF99';
+export const SHOP_GOLD = '#FFAA00';
+export const SHOP_GREEN = '#55FF55';
+export const shopYellow = (msg: string) => color(SHOP_YELLOW, text(msg));
+export const shopGold = (msg: string) => color(SHOP_GOLD, text(msg));
+export const shopGreen = (msg: string) => color(SHOP_GREEN, text(msg));
+
+addTranslation('shops.you_bought', {
+  fi_fi: 'Ostit %s kpl hintaan %s %s',
+  en_us: 'You bought %s items with %s %s',
+});
+
+addTranslation('shops.title', {
+  fi_fi: '----------Kauppa-arkku----------',
+  en_us: '-----------Shop-chest-----------',
+});
+
+addTranslation('shops.footer', {
+  fi_fi: '--------------------------------',
+  en_us: '--------------------------------',
+});
+
+addTranslation('shops.selling', {
+  fi_fi: 'Myydään',
+  en_us: 'Selling',
+});
+
+addTranslation('shops.bying', {
+  fi_fi: 'Ostetaan',
+  en_us: 'Buying',
+});
+
+addTranslation('shops.price', {
+  fi_fi: 'Hinta',
+  en_us: 'Price',
+});
+
+addTranslation('shops.unit_per_item', {
+  fi_fi: '%s / kpl',
+  en_us: '%s / item',
+});
+
+addTranslation('shops.tax_tooltip', {
+  fi_fi: 'Verottaja: %s \nVeron arvo: %s',
+  en_us: 'Tax collector: %s \nTax amount: %s',
+});
+
+addTranslation('shops.unknown', {
+  fi_fi: 'Tuntematon',
+  en_us: 'Unknown',
+});
+
+addTranslation('shops.VAT', {
+  fi_fi: 'Arvonlisävero',
+  en_us: 'Value-added tax',
+});
+
+addTranslation('shops.tax_free_price', {
+  fi_fi: `Veroton hinta: ${ChatColor.GOLD}%.2f${ChatColor.RESET} %s / kpl`,
```

Comment:
Using legacy formatting within "new" formatting (in this case, call to 'yellow') is possible not safe long-term. We should try to figure out alternatives, even though this is not *critical* yet.